lib/decoder.js

```diff
@@ -213,6 +224,10 @@ Decoder.prototype.decode = function(len, data, callback) {
     }
   }
 
+  if(len === 11) {
```

Comment:
No, there are other DPTs with 3 bytes payload (which should also result in len = 11): DPT10 and DPT (date, time), but they haven't been implemented yet, so there is no conflict right now. To be perfectly honest, it is impossible to tell the DPT by looking at the length, even with some additional validation.